api/qanpb/object_details.proto

```diff
@@ -65,5 +74,35 @@ message MetricValues {
   float max = 5;
   float avg = 6;
   float p99 = 7;
-  float p_total = 8;
+  float percent_of_total = 8;
+}
+
+// Labels are list of labels or dimensions values.
+message Labels {
+  repeated string value = 1;
+}
+
+// QueryExampleRequest defines filtering of query examples for specific value of
+// dimention (ex.: host=hostname1 or queryid=1D410B4BE5060972.
+message QueryExampleRequest {
+  google.protobuf.Timestamp period_start_from = 1;
+  google.protobuf.Timestamp period_start_to = 2;
+  string filter_by = 3; // dimention: queryid | host ...
```

Comment:
Please write comments above fields. That way they show up in generated code and Swagger docs.

Reply:
Fixed.